components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/cache/caches.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * API Cache
+ */
+const Cache = Java.type("org.eclipse.dirigible.components.api.cache.CacheFacade");
+
+export class Caches {
+
+    public static contains(key: string): boolean {
+        return (Cache.contains(key));
+    }
+
+    public static get(key: string): string | undefined {
```

Comment:
I think the return type should be `any | undefined` instead of `string | undefined`, as the `get()` method in the `CacheFacade` returns `Object`.